tensorboard/components/tf_dashboard_common/data-loader-behavior.ts

```diff
@@ -216,6 +223,8 @@ namespace tf_dashboard_common {
                 // Read-only property have a special setter.
                 this._setDataLoading(false);
               }
+
+              this._loadDataAsync = null;
```

Comment:
This should go in the `!result.cancelled` section?

Or if we guard every `cancelAsync()` call, we could drop this.

Reply:
Why? After the `async` is done, it should set the handle to null.